package.json

```diff
@@ -8,18 +8,8 @@
   "dependencies": {},
   "devDependencies": {
     "grunt": "*",
-    "grunt-autoprefixer": "*",
     "grunt-checktextdomain": "*",
-    "grunt-contrib-cssmin": "*",
-    "grunt-contrib-imagemin": "*",
-    "grunt-contrib-jshint": "*",
-    "grunt-contrib-uglify": "*",
-    "grunt-contrib-watch": "*",
-    "grunt-csscomb": "*",
-    "grunt-jsbeautifier": "*",
-    "grunt-jsvalidate": "*",
     "grunt-phplint": "*",
```

Comment:
@marksabbath Do we need grunt-phplint? It's flagged by `npm audit` as having vulnerabilities. Is it worth switching to the more up-to-date package.json contents from here instead https://github.com/studiopress/genesis-portfolio-pro/blob/develop/package.json

Reply:
IMHO we should migrate to PHPCS. If that would be the case, I'm more than happy to remove that package.

Reply:
Ok, @marksabbath, let's add it in the future since time is short. Everything else here looks good to me, and removing the unused styledocco was a good call!